Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -66,13 +65,11 @@ void InitSoundStream()
     g_sound_stream = std::make_unique<NullSound>();
     g_sound_stream->Init();
   }
+}
 
-  // Ideally these two calls would be done in AudioInterface::Init so that we don't
-  // need to have a dependency on AudioInterface here, but this has to be done
-  // after creating g_sound_stream (above) and before starting audio dumping (below)
-  g_sound_stream->GetMixer()->SetDMAInputSampleRate(AudioInterface::GetAIDSampleRate());
-  g_sound_stream->GetMixer()->SetStreamInputSampleRate(AudioInterface::GetAISSampleRate());
-
+void PostInitSoundStream()
+{
+  // This needs to be called after HW::Init where input sample rates are set
```

Comment:
This should probably mention `AudioInterface::Init`, should it? `HW::Init` isn't wrong (because it calls `AudioInterface::Init`), but it should only be a secondary mention due to the connection in `EmuThread`.
Thoughts?

Reply:
I didnt want to be too specific about `AudioInterface::Init` since #9600 also takes advantage of this which would make the comment no longer correct, so I guess I'll just update the comment in that PR as well